src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,19 @@
+<hibernate-configuration>
+  <session-factory>
+    <property name="connection.driver_class">com.mysql.cj.jdbc.Driver</property>
+    <!-- property name="connection.driver_class">com.mysql.jdbc.Driver</property -->
+    <property name="connection.url">jdbc:mysql://localhost:3306/list_film?serverTimeZone=UTC</property>
+    <property name="hibernate.dialect">org.hibernate.dialect.MySQLDialect</property>
+    <property name="connection.username">root</property>
+    <property name="connection.password">13572468TRee@@</property>
+    <property name="connection.pool_size">3</property>
+    <!--property name="dialect">org.hibernate.dialect.MySQLDialect</property-->
+    <property name="current_session_context_class">thread</property>
+    <property name="show_sql">true</property>
+    <property name="hbm2ddl.auto">create-drop</property>
+
+    <mapping class="mate.academy.hibernate.relations.model.Movie" />
+    <mapping class="mate.academy.hibernate.relations.model.Country" />
+    <mapping class="mate.academy.hibernate.relations.model.Actor" />
+  </session-factory>
+</hibernate-configuration>
```

Comment:
add newline whenever you see this symbol
![image](https://github.com/mate-academy/hibernate-relations-hw/assets/49767049/1038e02f-e47f-4674-aa91-0f50828bbf2e)

Reply:
In my project in Idea there is no such symbol

Reply:
Yeah, it's a GitHub feature to indicate the files that don't have the trailing new line, you can look in your PR to see it. In order to avoid it, make sure each file ends with a new line :)